app/components/card/CardWithArticle.tsx

```diff
@@ -0,0 +1,37 @@
+import AnimatedButton from "../buttons/AnimatedButton";
+import Card from "./Card";
+
+interface CardWithArticleProps {
+  image: string | null;
+  title: string;
+  linkToArticle: string;
+  createdAt: string;
+}
+
+const CardWithArticle = (props: CardWithArticleProps) => {
+  const { image, title, linkToArticle, createdAt } = props;
+
+  return (
+    <Card>
+      <>
+        <img
+          className="h-[15rem] w-full"
+          src={image ?? "/assets/default-article-image.svg"}
+        />
+        <h1 className="my-6 ml-4 text-left text-base">{title}</h1>
+
+        <footer className="flex items-center justify-between p-2 leading-none md:p-4">
+          <a href={linkToArticle} target="_blank">
```

Comment:
Fra ESLint:
`Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations`